base_address_parish/models/res_country.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Riverminds Cia Ltda - Mamfredy Mejia Matute
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Country(models.Model):
+    _inherit = "res.country"
+
+    enforce_parishes = fields.Boolean(
+        help="Check this box to ensure every address created in that country has a "
+        "Parish' chosen in the list of the city's parishes.",
+    )
```

Comment:
what is use for this field???
Field `parish_id  `is ever show on partner view, i think than attrs must be based on this field

Reply:
The parish concept is based for country. Each country define if have or not parishes

Reply:
my point is: field `enforce_parishes  `has not use on any view on `attrs` or python code

Reply:
I think that this field if you want keep it, should be implemented like

https://github.com/odoo/odoo/blob/a19d81bcab978150cf01f1ae85543adc5cb92e5e/addons/base_address_city/models/res_partner.py#L56

Reply:
The suggestion has added in the last commit today